Jellyfin.Plugin.AniList/Providers/AniList/ApiModel.cs

```diff
@@ -234,7 +234,18 @@ public List<PersonInfo> GetPeopleInfo()
         public List<string> GetTagNames()
         {
             PluginConfiguration config = Plugin.Instance.Configuration;
-            return (from tag in tags where config.AniListShowSpoilerTags || !tag.isMediaSpoiler select tag.name).ToList();
+
+            tags = tags.Where(o => o.rank >= config.MinTagRank).OrderBy(o => o.rank).ToList();
```

Comment:
`OrderBy` sorts in ascending order so you'll only keep the low ranked tags when you do `Take` later on